snyk-import/const.ts

```diff
@@ -1 +0,0 @@
-export const API = "https://api.snyk.io";
```

Comment:
Didn't want to comment on this, but I much prefer not having a separate file just for this.